# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/procps/hotfix.patch.musl
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- procps-3.3.17/w.c.vanilla	2021-10-20 19:48:07.910085423 +0200
+++ procps-3.3.17/w.c	2021-10-20 19:49:03.164083220 +0200
@@ -57,7 +57,6 @@
 #include <unistd.h>
 #ifdef HAVE_UTMPX_H
 #	include <utmpx.h>
-#else
 #	include <utmp.h>
 #endif
 #include <arpa/inet.h>
--- procps-3.3.17/proc/escape.c.vanilla	2021-10-20 19:47:47.326086244 +0200
+++ procps-3.3.17/proc/escape.c	2021-10-20 19:47:54.556085956 +0200
@@ -20,6 +20,7 @@
 #include <stdio.h>
 #include <sys/types.h>
 #include <string.h>
+#include <langinfo.h>
 #include <limits.h>
 #include "procps.h"
 #include "escape.h"
